internal/utils/utils.go

```diff
@@ -117,3 +125,22 @@ func GenStr(length int, charset string) string {
 	// Convert and return the generated string
 	return string(randomByteStr)
 }
+
+// IsURL Verify is input string is a valid HTTP(s) URL.
```

Comment:
You could replace the documentation comment with "IsURL verifies if an input string is a valid HTTP(s) URL." for better comprehension.